src/test/scala/nl.knaw.dans.easy.deposit/servlets/UploadSpec.scala

```diff
@@ -349,7 +349,8 @@ class UploadSpec extends DepositServletFixture {
     ) {
       absoluteTarget.list.size shouldBe 2
       status shouldBe CONFLICT_409
-      body shouldBe s"The following file(s) already exist on the server: some/3.txt, some/2.txt"
+      val prefix = "The following file(s) already exist on the server:"
+      body should (equal(s"$prefix some/3.txt, some/2.txt".toString) or equal(s"$prefix some/2.txt, some/3.txt".toString))
```

Comment:
isn't this a string already? Why use `.toString`?

Reply:
Some subtle difference made the test fail.